utils/perl5lib/Module/ModuleLoader.pm

```diff
@@ -655,15 +655,26 @@ sub loadSoftModules()
 
     my %newbuildenv;
 
+	# ndk: add a loop to see if our above module adjusting _removed_
```

Comment:
New CIME has a python module loader but I think the perl one is still used in code that hasn't been ported yet. It shouldn't be used much longer, so maybe we shouldn't worry.